donut/modules/newsgroups/helpers.py

```diff
@@ -41,7 +41,7 @@ def get_my_newsgroups(user_id, send=False):
     """
     if send:
         query += """
-            AND send = 1
+            AND (control = 1 OR send = 1)
```

Comment:
I don't think we should be special-casing this; `control` and `send` mean separate things. What was the reason for doing this?